blynklib_cp.py

```diff
@@ -196,7 +198,20 @@ def _get_socket(self):
             self._state = self.CONNECTING
             self._socket = socket.socket()
             self._socket.connect(socket.getaddrinfo(self.server, self.port)[0][4])
-            self._socket.settimeout(self.SOCK_TIMEOUT)
+            if (self.ssl_cert is not None):
+                self.log('Using SSL socket...')
+                import ssl, os
+                sslContext = ssl.create_default_context()
+                sslContext.verify_mode = ssl.CERT_REQUIRED
+                if (self.ssl_cert == "default"):
+                    caFile = os.path.dirname(__file__) + "/certificate/_blynk-cloudcom.crt"
```

Comment:
Please use "default" not as hardcode for blynk cloud certificate but as system’s default CA certificates 
It means  calling  ssl.create_default_context(caFile=None)
According this document https://docs.python.org/3/library/ssl.html